sale_automatic_workflow/__manifest__.py

```diff
@@ -0,0 +1,21 @@
+# © 2011 Akretion Sébastien BEAU <sebastien.beau@akretion.com>
+# © 2013 Camptocamp SA (author: Guewen Baconnier)
+# © 2016 Sodexis
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    'name': 'Sale Automatic Workflow',
+    'version': '11.0.1.0.0',
+    'category': 'Sales Management',
+    'license': 'AGPL-3',
+    'author': "Akretion,Camptocamp,Sodexis,Odoo Community Association (OCA)",
+    'website': 'http://www.akretion.com/',
```

Comment:
replace w/ `https://github.com/OCA/sale-workflow`

Reply:
@arkostyuk While I agree with you that it is better, I do not think that is correct convention, traditionally we have not changed original authors websites.

Reply:
AFAIR, the [convention](https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#modules) says that the `'website'` attribute should point to OCA repo for the sake of easing the access to the most relevant info about that module, though all of the author's credits (including website, I assume) shall remain in module's README :)

Reply:
@arkostyuk interesting, never seen that before.  Usually we try and change as little as possible on forward ports, but I'll change this one but I'll leave to you the PR's for prior versions